index.js

```diff
@@ -3,15 +3,25 @@ export default async function pReflect(promise) {
 		const value = await promise;
 
 		return {
+			status: 'fulfilled',
 			isFulfilled: true,
 			isRejected: false,
 			value
 		};
 	} catch (error) {
 		return {
+			status: 'rejected',
 			isFulfilled: false,
 			isRejected: true,
 			reason: error
 		};
 	}
 }
+
+export function isFulfilled(promiseResult) {
```

Comment:
These need a doc comment and also be documented in the readme.

Would also be useful to document that this is a workaround for https://github.com/microsoft/TypeScript/issues/32399

Reply:
PTAL